dracula-theme.el

```diff
@@ -56,7 +55,8 @@
    'dracula
    ;; default
    `(cursor ((,class (:background ,fg3))))
-   `(default ((,class (:background ,bg1 :foreground ,fg1))))
+   `(default ((((type nil)) (:background "#000000" :foreground ,fg1))
```

Comment:
From my test runs, it’s a fall-through semantics; first match wins. So from my tests, the evaluation goes:

*Frame shown on terminal*

- tries matching `type nil` against type `type nil` (the representing terminal frames)
- succeeds
- applies settings
- next condition (`,class…`) is not checked

*Frame shown on non-terminal*
- tries matching `type nil` aganist `type x`, `type w32`, etc.
- fails
- next condition is checked and applied

I tested this by setting only the background in the first condition, and only the foreground in the second condition.